app/Http/Requests/Album/SetAlbumCoverRequest.php

```diff
@@ -26,7 +26,7 @@ class SetAlbumCoverRequest extends BaseApiRequest implements HasAlbum, HasPhoto
 	 */
 	public function authorize(): bool
 	{
-		return Gate::check(AlbumPolicy::CAN_EDIT, [AbstractAlbum::class, $this->album]) && Gate::check(PhotoPolicy::IS_VISIBLE, $this->photo);
+		return Gate::check(AlbumPolicy::CAN_EDIT, [AbstractAlbum::class, $this->album]) && Gate::check(PhotoPolicy::CAN_SEE, $this->photo);
```

Comment:
Please keep it as it is right now, this is not a CR. For the moment we only change `IS_VISIBLE` to `CAN_SEE` and that's fine.

But whenever I see things like this, I feel the urge to come up with a proper photo right `CAN_EDIT`.